SECURITY.md

```diff
@@ -2,10 +2,20 @@
 
 ## Reporting a Vulnerability
 
-To report security issues please send an e-mail to [security@lightprotocol.com](mailto:security@lightprotocol.com).
+We appreciate any vulnerability found and responsibly disclosed.
+All reward amounts are determined based on our ongoing [bug bounty program](https://immunefi.com/bug-bounty/light-protocol/information/) with Immunefi.
 
-For sensitive information or critical issues, please contact the above e-mail address with 'CRITICAL' in the subject line and we will respond with a mechanism to securely communicate.
+### Smart Contract and Infrastructure
```

Comment:
```suggestion
### Smart Contract
```

Reply:
All bugbounties are for Solana programs, from our codebase only the forester or photon (photon is not directly our codebase though) would classify as infrastructure.